DataAccessLayer/Classes/Employee.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace ProductionDataAccessLayer.Classes
+{
+    public class Employee
+    {
+       /// <summary>
+       /// Employee's Identificator
+       /// </summary>
+        public int Id { get; set; }
+        /// <summary>
+        /// Employee's name
+        /// </summary>
+        public string Name { get; set; }
+        /// <summary>
+        /// Employee's email
+        /// </summary>
+        public string Email { get; set; }
+        /// <summary>
+        /// Employee's last name
+        /// </summary>
+        public string LastName { get; set; }
+        /// <summary>
+        /// Employee's password
+        /// </summary>
+        public string Password { get; set; }
+        /// <summary>
+        /// RoleId can be...
```

Comment:
This comment is uncompleted but I understand that it is because we don't have the data currently.
The next time you can add
`/// TODO: Update comment when the data is ready on data base`